en/config/actuators.md

```diff
@@ -79,6 +79,35 @@ The `X`, `Y`, `Z` positions are in [FRD coordinate frame, relative to the _centr
 Note, this may not be the same as the position of the flight controller!
 :::
 
+#### Geometry: Helicopter
+
+The geometry for a [Generic Helicopter (with Tail ESC)](../airframes/airframe_reference.md#copter_helicopter_generic_helicopter_(tail_esc)) is shown below.
+
+This is the only helicopter configuration supported, and allows for a single main rotor with any type of swash-plate and an ESC tail rotor.
+
+![Geometry: helicopter](../../assets/config/actuators/qgc_geometry_helicopter.png)
```

Comment:
Do we have an actuators screen for a typical helicopter setup (complete). Might be useful for a standalone helicopter actuators setup page.